compose/backend.yml

```diff
@@ -91,9 +91,9 @@ services:
       - service-auth-provider-api
 
   sidam-api:
-    image: hmcts.azurecr.io/hmcts/sidam/idam-api:1.0.0
+    image: "hmctspublic.azurecr.io/idam/api:pr-350"
```

Comment:
how come we are using a branch of idam?

Reply:
@mario-paniccia  , that's the new location for sidam image, and also as per SIDAM team we have to use PR for now, and will change to latest when they push proper tagging